src/components/pulse-dashboard/TestItem.tsx

```
@@ -113,46 +152,63 @@ export function TestItem({ test }: TestItemProps) {
             </AccordionTrigger>
             <AccordionContent className="pt-2 pl-2 pr-2 pb-1 bg-muted/30 rounded-lg">
               {test.errorMessage && (
                 <div className="mb-3">
-                  <h4 className="font-semibold text-xs text-destructive mb-1">Error:</h4>
+                  <h4 className="font-semibold text-xs text-destructive mb-1">
+                    Error:
+                  </h4>
                   <pre className="bg-destructive/10 text-xs p-2 rounded-md whitespace-pre-wrap break-all font-code max-h-20 overflow-y-auto">
-                    <span dangerouslySetInnerHTML={{ __html: ansiToHtml(test.errorMessage) }} />
+                    <span
+                      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)